jujugui/static/gui/src/app/components/entity-details/content/config-option/_config-option.scss

```diff
@@ -26,11 +25,12 @@
 
   &-description,
   &-default {
-    margin: .5rem 0 1rem 2rem;
+    margin: 0.5rem 0 1rem 2rem;
   }
 
   &-default {
     font-weight: normal;
-    font-family: 'Ubuntu Mono' ,'Consolas' ,'Monaco' ,'Lucida Console' ,'Courier New' ,Courier, monospace;
+    font-family: 'Ubuntu Mono', 'Consolas', 'Monaco', 'Lucida Console', 'Courier New', Courier,
```

Comment:
This is another one that is better all on one line, or each one on a line... 🤷‍♂️